src/types/EventListenerCallback.ts

```diff
@@ -0,0 +1,41 @@
+import { ImageStudioEventData } from './ImageStudioEventData';
+import { ImageStudioEventType } from './ImageStudioEventType';
+import { SDKEventType } from './SdkEventType';
+
+/**
+ * Callback function definition for Event Listeners
+ * @param data The Event Data object that was sent from Image Studio for this event.
+ * @returns a valid SDKEventType, or null to signify submitting the default configured response
+ */
+export type EventListenerCallback = (
+  data: ImageStudioEventData,
+) => SDKEventType | null;
+
+/**
+ * Interface for defining SDK response expectations
+ * @param default dictates the default SDKEventType to send to image-studio, a special null denotes we should not submit a response to image-studio.
+ * @param valid list of valid response types we can accept from custom callbacks.
+ */
+export interface EventListenerCallbackConfig {
+  defaultResponse: SDKEventType | null;
+  validResponses: SDKEventType[];
+}
```

Comment:
These aren't function params, so the documentation should go with the interface fields.

```suggestion
/**
 * Interface for defining SDK response expectations
 */
export interface EventListenerCallbackConfig {
  /**
   * Dictates the default SDKEventType to send to image-studio.
   * A special null denotes we should not submit a response to image-studio.
   */
  defaultResponse: SDKEventType | null;
  /**
   * List of valid response types we can accept from custom callbacks.
   */
  validResponses: SDKEventType[];
}
```

Reply:
Good spot, cheers 👍

Reply:
This PR had no other significant changes required, so we will save these for another time so this can be merged without re-review.